scripts/generateSlimMetadata.ts

```diff
@@ -0,0 +1,80 @@
+// Running this file with ~ yarn generate-slim-metadata will create slim metadata
+import {Api} from '../packages/api/src/Api';
+import MetadataVersioned from '@polkadot/metadata/Metadata/MetadataVersioned';
+
+import {createType} from '@cennznet/types';
+
+async function generateSlimMeta() {
+    // const provider = 'wss://cennznet.unfrastructure.io/public/ws'; // Use Azalea
```

Comment:
endpoint should be an option to pass in when run

Reply:
A comment explaining that when generating metadata we should try to connect to an endpoint which is running the desired version of cennznet.